content/v2.0/reference/design-principles.md

```diff
@@ -0,0 +1,38 @@
+---
+title: InfluxDB design principles
+description: >
+  Principles and tradeoffs related to InfluxDB design.
+weight: 7
+menu:
+  v2_0_ref:
+    name: Design principles
+v2.0/tags: [InfluxDB design principles]
+---
+
+InfluxDB 2.0 implements optimal design principles for time series data. Some of these design principles may have associated tradeoffs in performance.
+
+### Design principles and tradeoffs
+
+#### Data sent multiple times is duplicate data
+
+To simplify conflict resolution and increase write performance, InfluxDB assumes data sent multiple times is duplicate data.Duplicate data isn’t stored. In rare circumstances, data may be overwritten.
+
+#### Strict permissions to update and delete data
+
+To increase query and write performance, InfluxDB restricts access to update and delete data. Time series data is predominantly new data that is never updated. Deletes are almost always of data that isn't being written to, and contentious updates never occur.|Update and delete functionality is significantly restricted.
+
+#### Data ordered by timestamps in ascending order
+
+To significantly improve performance, data is written in time-ascending order. Writing points with random times (or in non-ascending order) is not performant.
+
+#### Queries first, consistency second
+
+Writing and querying the data is more important than having a strongly consistent view. Multiple clients can writes InfluxDB at high loads. Query returns may not include the most recent points if database is under heavy load.
```

Comment:
Should we just drop this one @sanderson @pierwill? Here's how I interpreted it:

If a read/write request comes in (during heavy load /transactions), InfluxDB prioritizes the client read/write request...if another transaction updates affected data after request has started processing, the response may not include the most recent points.

Reply:
Ok, I think I get what it's trying to say. Essentially, queries return whatever results exist at the time of query execution. If the ingest rate is high (multiple writes per ms), results may not include the _most_ recent data. InfluxDB will not "delay" a query to capture new points written since the time of execution.